install/prometheus.tmpl

```diff
@@ -8,42 +8,42 @@ global:
 scrape_configs:
   - job_name: 'prometheus'
     static_configs:
-      - targets: ['localhost:${PROMETHEUS_PORT:-9091}']
+      - targets: ['localhost:{{or .Prometheus.Port.Value "9091"}}']
 
   - job_name: 'node'
     static_configs:
       # node-exporter is on the host network so it can get access to the actual machine's network info
       # We have to use 'hosts.docker.internal' to refer to it due to this configuration
-      - targets: ['host.docker.internal:${EXPORTER_METRICS_PORT:-9103}']
+      - targets: ['host.docker.internal:{{or .ExporterMetricsPort.Value "9103"}}']
 
   - job_name: 'geth'
     static_configs:
-      - targets: ['${EC_HOSTNAME:-eth1}:${EC_METRICS_PORT:-9105}']
+      - targets: ['{{.GetExecutionHostname}}:{{or .EcMetricsPort.Value "9105"}}']
     metrics_path: /debug/metrics/prometheus
 
   - job_name: 'eth1'
     static_configs:
-      - targets: ['${EC_HOSTNAME:-eth1}:${EC_METRICS_PORT:-9105}']
+      - targets: ['{{.GetExecutionHostname}}:{{or .EcMetricsPort.Value "9105"}}']
```

Comment:
since we're using text/template now we could actually combine these into a single job, if you want

Reply:
(merged in HEAD to demonstrate)